src/BibleWell.App/App.axaml.cs

```diff
@@ -278,20 +280,26 @@ private ServiceProvider ConfigureServiceProvider(IConfiguration configuration, V
 
         // HTTP clients
         services
-            .AddHttpClient<IReadOnlyAquiferService, AquiferApiService>(client =>
-            {
-                client.BaseAddress = new Uri(configurationOptions.AquiferApiBaseUri);
-                client.DefaultRequestHeaders.UserAgent.Add(
-                    new ProductInfoHeaderValue(
-                        new ProductHeaderValue(
-                            Assembly.GetExecutingAssembly().GetName().Name ?? "",
-                            Assembly.GetExecutingAssembly().GetName().Version?.ToString())));
-                client.DefaultRequestHeaders.Add("api-key", configurationOptions.AquiferApiKey);
-            })
-            .SetHandlerLifetime(TimeSpan.FromMinutes(minutes: 5))
-            .AddPolicyHandler(GetRetryPolicy());
+            .ConfigureHttpClientDefaults(builder => builder
+                .SetHandlerLifetime(TimeSpan.FromMinutes(minutes: 5))
+                .AddStandardResilienceHandler()
```

Comment:
This updates from directly using Polly with retry policies to instead using the newer baked-in .net resilience handling (which uses Polly under hood).